test/parallel/test-repl.js

```diff
@@ -180,6 +180,13 @@ function error_test() {
       expect: prompt_multiline },
     { client: client_unix, send: '})()',
       expect: '1' },
+    // Multiline function call
+    { client: client_unix, send: 'function f(){}; f(f(1,',
+      expect: prompt_multiline },
+    { client: client_unix, send: '2)',
+      expect: prompt_multiline },
+    { client: client_unix, send: ')',
+      expect: 'undefined' },
```

Comment:
The last `send` closes off the outer function call and prints the output, so there's no `prompt_multiline` prepending the result.